jsk_baxter_robot/baxtereus/baxter-util.l

```diff
@@ -23,7 +23,7 @@
      (unless r ;;
        (format *error-output* "; failed for normal ik, starting from relaxed position~%")
        (let  ((current-coords (send (cadr (memq :move-target args)) :copy-worldcoords)))
-         (send-super* :inverse-kinematics current-coords :rotation-axis nil :avoid-nspace-gain 0.1 :avoid-weight-gain 0.1 :stop 200 :avoid-collision-distance avoid-collision-distance :debug-view nil args)
+         (send-super* :inverse-kinematics current-coords :rotation-axis nil :avoid-nspace-gain 0.1 :avoid-weight-gain 0.1 :stop 200 :avoid-collision-distance avoid-collision-distance :debug-view debug-view args)
```

Comment:
Do you think
```
+         (send-super* :inverse-kinematics current-coords :rotation-axis nil :avoid-nspace-gain 0.1 :avoid-weight-gain 0.1 :stop 200 :avoid-collision-distance avoid-collision-distance args)
```
works?

c.f. http://www.catb.org/~esr/writings/cathedral-bazaar/cathedral-bazaar/ar01s07.html

> 13. ``Perfection (in design) is achieved not when there is nothing more to add, but rather when there is nothing more to take away.''

Reply:
I print out `args` 
when  i do ` (send *baxter* :rarm :inverse-kinematics larm-coords :rotation-axis nil :revert-if-fail nil)`
It seems default key words is not in `args`

```lisp
(:move-target
 #<cascaded-coords #X6716570 :rarm-end-coords  708.945 522.301 180.305 / 1.412 0.248 1.838>
 :collision-avoidance-link-pair
 ((#<bodyset-link #X7468e70 right_wrist  676.253 317.87 232.676 / 2.916 -0.259 1.827> #<bodyset-link #X62a0d30 right_upper_forearm  524.483 -11.682 348.66 / 2.506 -0.134 1.908>)
  (#<bodyset-link #X7468e70 right_wrist  676.253 317.87 232.676 / 2.916 -0.259 1.827> #<bodyset-link #X629fc80 right_lower_elbow470.147 -93.057 382.67 / 0.982 0.335 -0.408>)
  (#<bodyset-link #X62a0d30 right_upper_forearm  524.483 -11.682 348.66 / 2.506 -0.134 1.908> #<bodyset-link #X6069b00 left_upper_forearm  366.781 666.893 465.687 / -1.314 0.49 -2.029>)
  (#<bodyset-link #X629fc80 right_lower_elbow  470.147 -93.057 382.67 / 0.982 0.335 -0.408> #<bodyset-link #X6069b00 left_upper_forearm  366.781 666.893 465.687 / -1.314 0.49 -2.029>)
  (#<bodyset-link #X7468e70 right_wrist  676.253 317.87 232.676 / 2.916 -0.259 1.827> #<bodyset-link #X6069b00 left_upper_forearm 366.781 666.893 465.687 / -1.314 0.49 -2.029>)
  (#<bodyset-link #X62a0d30 right_upper_forearm  524.483 -11.682 348.66 / 2.506 -0.134 1.908> #<bodyset-link #X62a4000 left_lower_elbow  282.406 622.397 506.082 / 0.485 0.401 -2.256>)
  (#<bodyset-link #X629fc80 right_lower_elbow  470.147 -93.057 382.67 / 0.982 0.335 -0.408> #<bodyset-link #X62a4000 left_lower_elbow  282.406 622.397 506.082 / 0.485 0.401 -2.256>)
  (#<bodyset-link #X7468e70 right_wrist  676.253 317.87 232.676 / 2.916 -0.259 1.827> #<bodyset-link #X62a4000 left_lower_elbow  282.406 622.397 506.082 / 0.485 0.401 -2.256>)
  (#<bodyset-link #X62a0d30 right_upper_forearm  524.483 -11.682 348.66 / 2.506 -0.134 1.908> #<bodyset-link #X7546a28 left_wrist 631.558 685.847 293.786 / -2.499 -0.307 -2.162>)
  (#<bodyset-link #X629fc80 right_lower_elbow  470.147 -93.057 382.67 / 0.982 0.335 -0.408> #<bodyset-link #X7546a28 left_wrist  631.558 685.847 293.786 / -2.499 -0.307 -2.162>)
  (#<bodyset-link #X7468e70 right_wrist  676.253 317.87 232.676 / 2.916 -0.259 1.827> #<bodyset-link #X7546a28 left_wrist  631.558 685.847 293.786 / -2.499 -0.307 -2.162>)
  (#<bodyset-link #X62a0d30 right_upper_forearm  524.483 -11.682 348.66 / 2.506 -0.134 1.908> #<bodyset-link #X742a020 screen  182.7 0.0 686.0 / 1.571 1.499e-15 1.751>)
  (#<bodyset-link #X629fc80 right_lower_elbow  470.147 -93.057 382.67 / 0.982 0.335 -0.408> #<bodyset-link #X742a020 screen  182.7 0.0 686.0 / 1.571 1.499e-15 1.751>)
  (#<bodyset-link #X7468e70 right_wrist  676.253 317.87 232.676 / 2.916 -0.259 1.827> #<bodyset-link #X742a020 screen  182.7 0.0 686.0 / 1.571 1.499e-15 1.751>))
 :link-list
 (#<bodyset-link #X62b53f0 right_upper_shoulder  64.027 -259.027 129.626 / 0.251 0.0 0.0>
  #<bodyset-link #X62b5c48 right_lower_shoulder  130.865 -241.889 399.976 / 0.251 -0.046 -1.571>
  #<bodyset-link #X658b450 right_upper_elbow  229.562 -216.583 404.708 / 1.795 0.519 1.517>
  #<bodyset-link #X629fc80 right_lower_elbow  470.147 -93.057 382.67 / 0.982 0.335 -0.408>
  #<bodyset-link #X62a0d30 right_upper_forearm  524.483 -11.682 348.66 / 2.506 -0.134 1.908>
  #<bodyset-link #X64fadb0 right_lower_forearm  658.498 206.847 261.118 / 1.412 0.248 0.267>
  #<bodyset-link #X7468e70 right_wrist  676.253 317.87 232.676 / 2.916 -0.259 1.827>)
 :rotation-axis
 nil
 :revert-if-fail
 nil)
```

Reply:
I updated.